helix-term/src/ui/editor.rs

```diff
@@ -1062,6 +1062,16 @@ impl Component for EditorView {
                     (Mode::Insert, Mode::Normal) => {
                         // if exiting insert mode, remove completion
                         self.completion = None;
+
+                        // For user friendly,
+                        // If we go from insert mode to normal node, and press no extra keys,
+                        // should try to make current line empty(restore pre-inserted indent).
+                        //
+                        // When we press no extra keys to normal mode, the `last_insert.1` just contains one key(ESC).
```

Comment:
It took me multiple times to read to understand this, can we please make this easier to understand?

Reply:
Ahh, sorry for my poor expression, I like this comment :)